<launch>

  <!-- Start openpose_ros -->
  <node pkg="openpose_ros" type="openpose_ros_node" name="openpose_ros_node" output="screen">
    <param name="image_topic"                value="/pepper_robot/camera/front/image_rect_color" />
    <param name="input_image_transport_type" value="compressed" />
    <param name="output_topic"               value="/openpose_ros/human_list" />
    <param name="display_output"             value="true" />
    <param name="print_keypoints"            value="false" />
    <param name="save_original_video"        value="false" />
    <param name="save_openpose_video"        value="false" />
    <param name="original_video_file_name"   value="original_video_output.avi" />
    <param name="openpose_video_file_name"   value="openpose_video_output.avi" />
    <param name="video_fps"                  value="10" />
  </node>

  <node pkg="pepper_demo" name="process_openpose_humans" type="process_openpose_humans.py" output="screen"> 
  </node>

  <node pkg="pepper_demo" name="pepper_demo" type="pepper_demo.py" output="screen"> 
  </node>


</launch>